Userland/Libraries/LibWeb/UIEvents/MouseEvent.idl

```diff
@@ -12,6 +12,8 @@ interface MouseEvent : UIEvent {
     readonly attribute double y;
     readonly attribute double pageX;
     readonly attribute double pageY;
+    readonly attribute double movementX;
+    readonly attribute double movementY;
```

Comment:
Can you link the PointerLock spec here above the movementX and movementY? 
 https://w3c.github.io/pointerlock/#dom-mouseevent-movementx
 
 We also need to make sure that the MouseEventInit dictionary matches the spec definition for the extensions from this spec, and the base UIEvents spec. That's how JS can create synthetic events, which is a real use case.